pontoon/base/views.py

```diff
@@ -440,9 +444,13 @@ def entities(request):
     except (MultiValueDictKeyError, ValueError) as err:
         return HttpResponseBadRequest('Bad Request: {error}'.format(error=err))
 
-    project = get_object_or_404(Project, slug=project)
     locale = get_object_or_404(Locale, code__iexact=locale)
 
+    if project == 'all-projects':
+        project = Project(slug=project)
```

Comment:
I'm kinda worried that at some point some part of the code will mutate (e.g. .save) this object.Maybe we should create a separate instance of project that would be loaded in case when slug is all-projects? That would decrease the risk of spawning multiple new projects in future and would be less additional conditions on backend side.